### PR TITLE
Improve Dev onboarding docs

### PR DESCRIPTION
**Pull request by luisramos0**
Making GETTING STARTED and CONTRIBUTING a bit simpler.

I also made the wiki menu more simple:
image

And finally I managed to merge the 3 OSX setup pages into one :tada: 
https://github.com/openfoodfoundation/openfoodnetwork/wiki/Development-Environment-Setup%3A-OS-X

I think the create db user step is still repeated between GETTING STARTED and the OSX setup page but I am not sure there are differences so I left it as is.

### CODE REVIEW

File: GETTING_STARTED.md [hunk 1]
```diff
@@ -52,14 +48,12 @@ This will create the "ofn" user as superuser and allowing it to create databases
...
-
 Now, your dreams of spinning up a development server can be realised:
 
     bundle exec rails server
 
+Go to [http://localhost:3000](http://localhost:3000) to play around!
+
 To login as the default user, use:
 
     email: ofn@example.com
```
Comment (mbudm) [inline GETTING_STARTED.md:59]: Not sure if it's just me but I found that `spree@example.com|spree123` worked instead of this credential.

Comment (luisramos0) [reply GETTING_STARTED.md:59]: It was recently changed, you must have had a version of master with the previous version.

Comment (Matt-Yorkley) [reply GETTING_STARTED.md:59]: If your environment was set up before the change, it won't be overwritten

Comment (luisramos0) [reply GETTING_STARTED.md:59]: ah, yes, until the first db:reset :+1:

Comment (mbudm) [reply GETTING_STARTED.md:59]: Ace

**Comment by sauloperez** [comment]
Was this weird (md) in the wiki's sidebar intended @luisramos0 ?

Screenshot from 2020-06-03 09-49-21

**Comment by luisramos0** [comment]
I didnt change that part Pau. I think the intention is to link contributing to the md in the repo yes.

**Pull request merged**